internal/db/db.go

```diff
@@ -0,0 +1,68 @@
+package db
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/canonical/chisel/internal/jsonwall"
+	"github.com/klauspost/compress/zstd"
+)
+
+const schema = "0.1"
+
+// New creates a new Chisel DB writer with the proper schema.
+func New() *jsonwall.DBWriter {
+	options := jsonwall.DBWriterOptions{Schema: schema}
+	return jsonwall.NewDBWriter(&options)
+}
+
+func dbPath(root string) string {
+	return filepath.Join(root, ".chisel.db")
```

Comment:
I remember Cristovão suggested we use the /var/lib/chisel path (probably chisel.db inside it?), which may be reasonable since this is leveraging standard Ubuntu packages which will be using proper LSB paths anyway, so we may as well respect it instead of pretending that we'll have a mostly empty tree.